packages/admin/admin/src/error/errordialog/ErrorDialog.tsx

```diff
@@ -84,3 +70,123 @@ export const ErrorDialog: React.FunctionComponent<ErrorDialogProps> = ({ show =
+            <Typography variant="h6">
+                <FormattedMessage
+                    id="comet.errorDialog.defaultUserMessage"
+                    defaultMessage="The following {errorCount, plural, one {error} other {errors} } {errorCount, plural, one {has} other {have} } occured:"
+                    values={{ errorCount: error.length }}
+                />
+            </Typography>
+            <ErrorList>
+                {error.map((error) => (
+                    <ErrorListItem key={error}>{error}</ErrorListItem>
+                ))}
+            </ErrorList>
+            <Typography gutterBottom>
+                <FormattedMessage
+                    id="comet.errorDialog.copyToClipboardInstruction"
+                    defaultMessage="This information may prove useful for debugging the {errorCount, plural, one {error} other {errors} }. Please copy it and send it to your administrator."
+                    values={{ errorCount: error.length }}
+                />
+            </Typography>
+            <CopyToClipboardButton copyData={copyData} />
+            {additionalInformation && (
+                <>
+                    <Divider sx={{ marginTop: 4, marginBottom: 4 }} />
+                    <Typography component="p" variant="caption">
+                        <strong>
+                            <FormattedMessage id="comet.errorDialog.details" defaultMessage="Error details" />
+                        </strong>
+                    </Typography>
+                    <Typography component="p" variant="caption">
+                        <FormattedMessage id="comet.errorDialog.details.errorType" defaultMessage="Type" />:{" "}
+                        {errorTypeLabels[additionalInformation.errorType]}
+                    </Typography>
+                    <Typography component="p" variant="caption">
+                        <FormattedMessage id="comet.errorDialog.details.httpStatus" defaultMessage="HTTP status" />:{" "}
+                        {additionalInformation.httpStatus}
+                    </Typography>
+                    <Typography component="p" variant="caption">
+                        <FormattedMessage id="comet.errorDialog.details.url" defaultMessage="URL" />: {additionalInformation.url}
+                    </Typography>
+                    <Typography component="p" variant="caption">
+                        <FormattedMessage id="comet.errorDialog.details.timestamp" defaultMessage="Timestamp" />: {additionalInformation.timestamp}
+                    </Typography>
+                </>
+            )}
+        </>
+    );
+}
+
+const ErrorList = styled(List)`
+    list-style-type: disc;
+    padding-inline-start: ${({ theme }) => theme.spacing(6)};
+`;
+
+const ErrorListItem = styled(ListItem)`
+    display: list-item;
+    padding-left: 0;
+`;
+
+type CopyToClipboardButtonProps = {
+    copyData: string;
+};
+
+function CopyToClipboardButton({ copyData }: CopyToClipboardButtonProps): JSX.Element {
```

Comment:
@thomasdax98 we already have a CopyToClipboardButton (introduced by @jamesricky), but it is not flexible enough for this use case (always an IconButton, no text). Maybe we could discuss the button with UX and then create a generic component for all use cases?